prov/efa/src/rdm/rxr_msg.c

```diff
@@ -699,6 +699,16 @@ struct rxr_op_entry *rxr_msg_alloc_unexp_rx_entry_for_tagrtm(struct rxr_ep *ep,
 	return rx_entry;
 }
 
+/**
+ * @brief Split a posted rx entry to be consumed by multi-recv buffers.
+ * After the split, the iov of the posted entry will be adjusted to
+ * reflect the consumption.
+ * @param[in] ep endpoint
+ * @param[in/out] posted_entry the rx entry posted by user for multi-recv
```

Comment:
Does your ide highlight this syntax?
On my side
```
@param[in,out]
```
is highlighted, but
```
@param[in/out]
```
is not.

Reply:
Yes, it should be a comma. Updated, thanks.